README.md

```diff
@@ -180,6 +180,7 @@ paige:
       account_id: ""
     counter: # See https://counter.dev
       account_id: ""
+      utc_offset: "" # GMT +/- local timezone
```

Comment:
GMT is different than UTC (https://en.wikipedia.org/wiki/UTC_offset). We should also document that it should be just the hour integer without a leading zero and no minutes.

Perhaps something like `# UTC offset; examples are "-7", "0", and "7"`?

Reply:
(I'm assuming that "0" and "7" are correct.)